tests/ping6_clicker.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2017
+#
+# All rights reserved.
+#
+# This file is distributed under the Clear BSD license.
+# The full text can be found in LICENSE in the root directory.
+
+import rootfs_boot
+from devices import board
+
+class BoardPing6ClickerDev(rootfs_boot.RootFSBootTest):
+	'''Board lowpan interface can ping6 the clicker.'''
+	def runTest(self):
+	# Make Lan-device ping Wan-Device
+		board.sendline('\nping6 -I lowpan0 -c 20 fe80:0000:0000:0000:0204:a30b:0000:c184')
```

Comment:
We have seen different issues depending on the size of the packets so it would be useful for this test to ping 20 packets of small, medium and large sizes. @mtusnio any suggestions on good sizes, I'm thinking along the lines of the fragmentation limit?

Also should the IP address to ping should come from an env var on the boardfarm slave?